policyengine_uk/variables/gov/hmrc/capital_gains_tax/responses.py

```diff
@@ -0,0 +1,168 @@
+from policyengine_uk.model_api import *
+from policyengine_core.simulations import *
+
+
+class relative_capital_gains_tax_change(Variable):
```

Comment:
```suggestion
class relative_capital_gains_mtr_change(Variable):
```
Could otherwise be confused for average tax rate